pkg/kubelet/apis/cri/services.go

```diff
@@ -43,6 +43,8 @@ type ContainerManager interface {
 	ListContainers(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error)
 	// ContainerStatus returns the status of the container.
 	ContainerStatus(containerID string) (*runtimeapi.ContainerStatus, error)
+	// UpdateContainerResources updates the cgroup resources for the container.
+	UpdateContainerResources(containerID string, resources *runtimeapi.LinuxContainerResources) error
```

Comment:
`LinuxContainerResources` are really the only API object that makes sense here.  The function name mirrors that resource.  What else about the container would you want to be able to change while the container was running?  Knowing that the runtime would need to support any such update as well.